LICENCE

```diff
@@ -0,0 +1,20 @@
+Copyright (c) 2011 HM Government
```

Comment:
There's an email going out about this soon, but this should be:

`Copyright (c) (year) Crown Copyright, rather than `HM Government`.

cc @annashipman